website/docs/visual-testing/reporter.md

```diff
@@ -0,0 +1,103 @@
+---
+id: visual-reporter
+title: Visual Reporter
+---
+
+The Visual Reporter is a new feature introduced in the `@wdio/visual-service`, starting from version [v5.2.0](https://github.com/webdriverio/visual-testing/releases/tag/%40wdio%2Fvisual-service%405.2.0). This reporter allows users to visualize the JSON diff reports generated by the Visual Testing service and transform them into a human-readable format. It helps teams better analyze and manage the visual testing results by providing a graphical interface for reviewing the output.
+
+To utilize this feature, ensure you have the required configuration to generate the necessary `output.json` file. This document will guide you through setting up, running, and understanding the Visual Reporter.
+
+# Prerequisites
+
+Before using the Visual Reporter, make sure you have configured the Visual Testing service to generate JSON report files:
+
+```ts
+export const config = {
+    // ...
+    services: [
+        [
+            "visual-regression",
```

Comment:
Is this the right reporter? Shouldn't it be
```suggestion
            "visual",
```

Reply:
OMG, this is really bad, thanks @christian-bromann !